tests/xharness/TestProject.cs

```diff
@@ -94,6 +94,14 @@ public TestProject GetTodayExtension ()
 			}
 		}
 
+		public bool IsMonotouch => Name.Contains ("monotouch");
```

Comment:
will the case ever change and will there ever be naming conflicts that would break this?
For example `monotouchless`

Reply:
No, but in that case, it would not hurt us. Will just run the two tests in the same job.